deploy/crds/org_v1_che_cr.yaml

```diff
@@ -16,6 +16,7 @@ metadata:
   name: eclipse-che
 spec:
   server:
+    useServiceHostNames: false
```

Comment:
Why is it `false` here while it is `true` in PR https://github.com/eclipse/che-operator/pull/507/files#diff-c9772ddc1ed08d9f54fa301ae58047756e33f73e5309795df6dca62d80411e51R19 ?

Reply:
This pr was created earlier. And initially we thought about default `false`. But looks like we can use true value to detect any `possible` regression faster. @tolusha @davidfestal Do you agree?

Reply:
Turned "on" this option. Related mail list here: https://www.eclipse.org/lists/che-dev/msg03934.html . If we will get bad feed back for this default I'll move it back to false.